pkg/artifactory/provider.go

```diff
@@ -65,6 +65,8 @@ func Provider() *schema.Provider {
 			"artifactory_keypair":                    resourceArtifactoryKeyPair(),
 			"artifactory_local_repository":           resourceArtifactoryLocalRepository(),
 			"artifactory_local_nuget_repository":     resourceArtifactoryLocalNugetRepository(),
+			"artifactory_local_maven_repository":     resourceArtifactoryLocalMavenRepository(),
```

Comment:
Actually, don't do this. These are identical in structure. Please create a generic function something like this:
`resourceArtifactoryLocalJava("gradle")` and `resourceArtifactoryLocalJava("maven")`

Reply:
that's not possible to pass a key and bool parameter without pretty big refactoring. I'll keep two resources

Reply:
I agree with @chb0github. Please refactor to use common function. The acceptance tests are your friend.